custom_components/trakt_tv/schema.py

```diff
@@ -8,6 +8,7 @@
 
 from .const import DOMAIN, LANGUAGE_CODES
 from .models.kind import BASIC_KINDS, NEXT_TO_WATCH_KINDS, TraktKind
+from .models.kind import ANTICIPATED_KINDS
```

Comment:
Move ANTICIPATED_KINDS up to the same import as others from .models.kind. 
Also, you're importing them but haven't defined them there yet (see my previous comment on ANTICIPATED_KINDS)